group_vars/geniza/vars.yml

```diff
@@ -24,24 +24,19 @@ wsgi_path: "{{ app_name }}.wsgi"
 python_version: "python3.6"
 # Override clone root to use deploy user home instead of root
 clone_root: "/home/{{ deploy_user }}/repos"
-# for Geniza, do we want to differentiate between qa/prod paths?
+# for Geniza, we don't distinguish between qa/prod paths
 install_root: '/srv/www/{{ app_name }}'
+# path to dir that holds local_settings.py
+template_path: "{{ app_name }}"
+solr_core: 'geniza'
 
 # override default production contexts (no code coverage for this project yet)
 deploy_contexts:
     - 'continuous-integration/travis-ci/push'
 
-app_vars:
-  - name: GENIZA_SOLR_URL
-    value: "{{ solr_url }}"
-  - name: GENIZA_SOLR_CORE
-    value: "{{ solr_collection }}"
-  - name: FLASK_APP
-    value: 'scripts/server.py'
-
```

Comment:
Do we not need these anymore?

Reply:
Oh, right! I had thought "we don't need environment variables, because everything's in `local_settings.py`" but I had forgotten about `FLASK_APP`. I'll push the fix now.